### PR TITLE
use calicoctl-util to apply manifests

### PR DESCRIPTION
**Pull request by krishgobinath**
## Description
1) Calicoctl docker image is based out of the scratch image,  hence it doesn't have any `Linux utils` or `shell`. Replace shell dependency with calicoctl-util tool

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit f0d9ef4b244a692e9ab9d167347f89e37a6660be

https://deploy-preview-3858--calico.netlify.app

**Comment by bcreane** [comment]
@krishgobinath - what tests have you run with this PR?

File: _includes/charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml [hunk 1]
```diff
@@ -54,13 +54,10 @@ spec:
             - name: DATASTORE_TYPE
               value: kubernetes
           command:
-            - /script/install.sh
+            - /calicoctl-util apply /calico-resources
```
Comment (doublek) [inline _includes/charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml:57]: @krishgobinath this is missing a `-f`.

Comment (doublek) [reply _includes/charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml:57]: Nevermind. This is not calicoctl.

Comment (doublek) [inline _includes/charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml:57]: This should be
```
          command:
            - /calicoctl-util
          args:
            - apply
            - /calico-resources
```

Comment (doublek) [reply _includes/charts/tigera-operator/templates/tigera-operator/02-tigera-operator.yaml:57]:
```suggestion
            - /calicoctl-util
          args:
            - apply
            - /calico-resources
```

**Comment by tmjd** [comment]
This depends on https://github.com/projectcalico/calicoctl/pull/2158
Casey and I have commented on that PR that we thought this was going to be done by adding a new feature to calicoctl to apply resources in a directory.

**Comment by caseydavenport** [comment]
Closing in favor of this guy: https://github.com/projectcalico/calico/pull/3874

**Pull request closed without merging**